src/module/custom.rs

```diff
@@ -345,4 +357,42 @@ impl ModuleCustomSections {
             }
         })
     }
+
+    /// Remove a custom section (by type) from the module.
+    ///
+    /// If there are multiple custom sections of the type `T` only the first one
+    /// is removed.
```

Comment:
"first" is a bit loosely defined, since the arena's items can't really be reordered. ah well.